.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:11.0.4-jdk-stretch
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-gradle-{{ checksum "build.gradle" }}-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+      - run:
+          name: build
+          command: ./gradlew clean build check
+      - save_cache:
+          paths:
+            - ~/.gradle
+          key: v1-gradle-{{ checksum "build.gradle" }}-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+          when: always
+      - run:
+          name: release
+          command: ./gradlew ciPerformRelease -s
```

Comment:
Doesn't this need the TRAVIS_BRANCH variable?

Reply:
Added.